package.json

```diff
@@ -35,7 +36,6 @@
     "np": "^6.2.1"
   },
   "dependencies": {
-    "@any-sender/client": "^0.2.0-beta.0",
     "@chainlink/contracts": "0.0.3",
```

Comment:
Check each of these and the devDependencies, I think some of them arent used. I would reccommend you find a tool to help you do this, so that it'll be much quicker in future

Reply:
Still some references to unused dependencies

Reply:
only one i can see not being used is chainlink and thus removed (dunno how that ended up in there at all? lol)

Reply:
You're still missing ts-mockito, do:
```
npm i -g depcheck
depcheck
```

Reply:
fixed